src/parser/shared/modules/spells/bfa/azeritetraits/GraceOfTheJusticar.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS/index';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+
+/**
+ * Grace of the Justicar
+ * Judging a foe heals up to 10 allies within 8 yards of that enemy for 899. (@iLvl 400)
+ * Example Log: https://www.warcraftlogs.com/reports/kMbVanmJwCg7WrAz#fight=last&type=summary&source=2
+ */
+class GraceOfTheJusticar extends Analyzer {
+  graceHealing = 0;
```

Comment:
```suggestion
  healing = 0;
```

The module is already called GraceOfTheJusticar